driver-core/src/main/java/com/datastax/driver/core/Connection.java

```diff
@@ -536,46 +534,13 @@ public String toString() {
 
     public static class Factory {
 
-        private static final Class[] EVENT_GROUP_ARGUMENTS = {int.class, ThreadFactory.class};
```

Comment:
I moved all of this to a new NettyUtil class to avoid having this code executed (static block) even if the client is supplying his custom EventLoopGroup.